### PR TITLE
eliminate probin for Detonation

### PR DESCRIPTION
**Pull request by zingale**
## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

File: Exec/science/Detonation/inputs-collision [hunk 1]
```diff
@@ -53,6 +51,11 @@ amr.refine.dengrad_rel.max_level = 0
 amr.refine.dengrad_rel.relative_gradient = 0.5
 amr.refine.dengrad_rel.field_name = density
 
+amr.refine.dxnucerr_rel.max_level = 20
```
Comment (maxpkatz) [inline Exec/science/Detonation/inputs-collision:54]: This has a typo, but you can also just delete this line.

File: Exec/science/Detonation/inputs-collision [hunk 2]
```diff
@@ -73,3 +76,51 @@ castro.small_plot_per_is_exact = 1
...
+
+problem.grav_acceleration = 1.1e8
+
+problem.use_early_stopping = 1
+problem.T_stop_cutoff = 4.e9
+problem.ts_te_stop_cutoff = 1.e200
+
+# Microphysics
+
+network.small_x = 1.d-12
```
Comment (maxpkatz) [inline Exec/science/Detonation/inputs-collision:106]: d

File: Exec/science/Detonation/inputs-collision (hunk 1 above)
Comment (zingale) [reply Exec/science/Detonation/inputs-collision:54]: fixed

File: Exec/science/Detonation/inputs-collision (hunk 2 above)
Comment (zingale) [reply Exec/science/Detonation/inputs-collision:106]: fixed

**Review by maxpkatz — APPROVED** [review]

**Pull request merged**